bitnami/elasticsearch/Chart.yaml

```
@@ -25,4 +25,4 @@ name: elasticsearch
 sources:
   - https://github.com/bitnami/bitnami-docker-elasticsearch
   - https://www.elastic.co/products/elasticsearch
-version: 17.1.8
+version: 17.2.8
```

`17.2.0` would be the next minor.

```suggestion
version: 17.2.0
```